lib/3scale/backend/transactor/status.rb

```diff
@@ -211,11 +194,40 @@ def with_report_and_hierarchy(reports)
           end
         end
 
-        def add_plan(xml, tag, plan_name)
+        def add_plan_section(xml, tag, plan_name)
           xml << "<#{tag}>"
           xml << plan_name.to_s.encode(xml: :text) << "</#{tag}>"
         end
 
+        def add_authorize_section(xml)
+          xml << '<authorized>'.freeze
+          if authorized?
+            xml << 'true'.freeze << '</authorized>'.freeze
+          else
+            xml << 'false'.freeze << '</authorized>'.freeze
+            xml << '<reason>'.freeze
```

Comment:
here you can do `xml << '<authorized>false</authorized><reason>'.freeze`

Reply:
the same principle applies to other places in the added methods

Reply:
:+1: . Done. In some other places I left it the way it is for readability